qutebrowser/browser/webview.py

```diff
@@ -487,12 +487,25 @@ def search(self, text, flags):
         old_scroll_pos = self.scroll_pos
         flags = QWebPage.FindFlags(flags)
         found = self.findText(text, flags)
+        backward = int(flags) & QWebPage.FindBackward
```

Comment:
Is the `int()` here needed? At least below, `flags & QWebPage.HighlightAllOccurrences` seems to work fine.